lib/data-types.js

```diff
@@ -531,9 +531,15 @@ VIRTUAL.prototype.key = VIRTUAL.key = 'VIRTUAL';
  */
 var ENUM = function(value) {
   var options = typeof value === 'object' && !Array.isArray(value) && value || {
-    values: Array.prototype.slice.call(arguments).reduce(function(result, element) {
-      return result.concat(Array.isArray(element) ? element : [element]);
-    }, [])
+    values: (function (args) {
+      var tmp = new Array(args.length);
```

Comment:
Makes the code a lot less readable for something that is only used once ever in an application.